dotnet/samples/Concepts/Caching/SemanticCachingWithFilters.cs

```diff
@@ -199,7 +199,7 @@ public async Task OnPromptRenderAsync(PromptRenderContext context, Func<PromptRe
             await collection.CreateCollectionIfNotExistsAsync();
 
             // Search for similar prompts in cache.
-            var searchResults = await collection.VectorizedSearchAsync(promptEmbedding, new() { Top = 1 }, context.CancellationToken);
+            var searchResults = await collection.VectorizedSearchAsync(promptEmbedding, top: 1, cancellationToken: context.CancellationToken);
```

Comment:
It's (slightly) annoying to have to specify the cancellationToken named argument just because we don't need to specify the options :/

Should we add an overload that doesn't accept the options, and have a default interface implementation - for .NET 8.0 only - that calls into the main one? We should try to not let the .NET Framework support cause an inferior dev experience to people using modern .NET...

@westey-m @adamsitnik what do you think?

Reply:
Opened #11382.